ebpf/go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/grafana/pyroscope/ebpf
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.9
```

Comment:
To be honest I don't really know what this change does and why is this needed. Do we need tolchain in main go.mod in the root of the repo?

Reply:
I don't understand it either, it seems to be coming in everytime I touch the `go get`.

```
git ls-files '**/go.mod' '**/go.work'  | xargs sed -i '/toolchain go/d'
```